language_service/src/completion.rs

```diff
@@ -19,11 +19,49 @@ const PRELUDE: [&str; 3] = [
     "Microsoft.Quantum.Intrinsic",
 ];
 
+struct IndentMaker<'a> {
+    contents: &'a str,
+    offset: u32,
+}
+
+impl<'a> IndentMaker<'a> {
+    fn new(compilation: &'a Compilation, source_name: &'_ str) -> Self {
```

Comment:
Surprised you need a lifetime for `source_name: &'_ str` . What happens if you remove it?

Reply:
I removed the `IndentMaker` in favor of using a single function.